### PR TITLE
Centralise CultureInfo

### PR DESCRIPTION
**Pull request by BaronGreenback**
Noticed we had numerous creations of CultureInfo of en-US.

This PR centralised to a single instance in similar to that of json, and should save some memory as most were static.

### CODE REVIEW

File: MediaBrowser.Common/Culture/CultureDefault.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    /// <summary>
+    /// Helper class for culture info across the codebase.
+    /// </summary>
+    public static class CultureDefault
+    {
+        /// <summary>
+        /// US Culture Info.
+        /// </summary>
+        public static readonly CultureInfo USCulture = CultureInfo.ReadOnly(new CultureInfo("en-US"));
```
Comment (crobibero) [inline MediaBrowser.Common/Culture/CultureDefault.cs:18]:
```suggestion
        public static readonly CultureInfo UsCulture = CultureInfo.ReadOnly(new CultureInfo("en-US"));
```
Analyzers will probably request the naming change

**Review by crobibero — APPROVED** [review]

**Review by Ullmie02 — APPROVED** [review]

**Comment by Bond-009** [comment]
99% of these could just use `InvariantCulture`

**Comment by BaronGreenback** [comment]
> 99% of these could just use `InvariantCulture`

InvariantCulture for numerics and Us for dates?

**Comment by Bond-009** [comment]
> InvariantCulture for numerics and Us for dates?

`InvariantCulture` *should* be fine for ints, not sure about floating point

**Comment by Bond-009** [comment]
Superseded by #6616

**Pull request closed without merging**